config/locales/en.yml

```diff
@@ -328,6 +330,7 @@ en:
       created_by_unknown: Unknown
       name: Group name
     index:
+      active_empty_message: You're not in any active groups.
```

Comment:
Very minor but we try to use curly quotes in translations:
```suggestion
      active_empty_message: You’re not in any active groups.
```

Reply:
Thanks! Updated